projects/igniteui-angular/src/lib/progressbar/linearbar.component.spec.ts

```diff
@@ -475,6 +474,30 @@ describe('IgLinearBar', () => {
             fix.detectChanges();
             expect(bar.classList.contains(INDETERMINATE_CLASS)).toEqual(true);
         });
+
+        it('Check if current value in % changes dynamically when max is increased', fakeAsync(() => {
+            const fix = TestBed.createComponent(InitLinearProgressBarComponent);
+            fix.detectChanges();
+
+            const bar = fix.componentInstance.linearBar;
+            let maxVal = 100;
+            const val =  50;
+
+            bar.max = maxVal;
+            bar.value = val;
+            let valueInPercent = Common.calcPercentage(bar.value, bar.max);
+
+            tick(tickTime);
+            fix.detectChanges();
+            expect(valueInPercent).toBe(50);
```

Comment:
Here, we calculated the expected valueInPercent on line 488. Now, we should compare this to the `valueInPercent` holded by the LinearBar itself:

typescript```
expect(valueInPercent).toBe(bar.valueInPercent);
```